README.md

```diff
@@ -86,6 +85,7 @@ A curated list of anime streaming/downloading websites.
 
  * [animebytes.tv](https://animebytes.tv/)
  * [animetorrents.me](https://animetorrents.me/)
+ * [bakabt.me](https://www.bakabt.me/)
```

Comment:
What's this? Seems to be asking a username and pass

Reply:
Private torrent tracker for anime. You can request invitation from irc channel. They will ask a few questions and proof for ratio from other private trackers. [https://wiki.bakabt.me/index.php/Sign_up](https://wiki.bakabt.me/index.php/Sign_up)

Reply:
Ah cool! Maybe we should add this info as well as the users you are landing on the page won't have this info.

Reply:
Just add a note near the link and it should be good